### ISSUE TITLE
deal with Mandrill changes

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#515]
> Going forward, all Mandrill users will be required to have a paid monthly MailChimp account and verify ownership of all sending domains. Here's the timeline: Starting 3/16, all new Mandrill users will create accounts through MailChimp, and current Mandrill users can merge their existing Mandrill account with a monthly MailChimp account. Current users will have until 4/27 to merge their accounts.

http://blog.mailchimp.com/important-changes-to-mandrill/

**Comment by whit537** [comment]
Basically, they're saying to use Amazon SES if you want cheap email.

**Comment by whit537** [comment]
> As they say, “culture eats strategy for breakfast,” so no matter how compelling or strategic it may seem to conquer the utility transactional space, MailChimp’s cultural DNA compels us to innovate on e-commerce personalization.

Hadn't heard that one before.

**Comment by whit537** [comment]
> Basically, they're saying to use Amazon SES if you want cheap email.

Actually, they're actually saying this:

> Startup developers looking for a cheap, reliable transactional service may want to consider Amazon SES.

:-)

**Comment by whit537** [comment]
Bottom line (from their email):

>If you don't have a paid monthly MailChimp account merged with your Mandrill account by 4/27, then your Mandrill account will be disabled.

**Comment by whit537** [comment]
https://gratipay.freshdesk.com/helpdesk/tickets/4171

**Comment by rohitpaulk** [comment]
+1 on switching to SES

**Comment by clone1018** [comment]
Some alternatives:
https://www.mailgun.com/
https://sendgrid.com/

**Comment by clone1018** [comment]
MailGun looks just like what Mandrill was

**Comment by rohitpaulk** [comment]
Switching to SES => Far less chances of us having to make changes because an email provider decided to change course

**Comment by whit537** [comment]
We've got a lot more fun/important stuff to work on, so what's the *easiest* change? Probably following Mandrill/MailChimp's migration path to a paid MailChimp account. No?

**Comment by clone1018** [comment]
Does this make sense in Gratipay's context? http://www.dangrossman.info/2016/02/28/mandrills-betrayal/

"You can no longer use it to send the same mail to multiple people, like an alert or notification."

**Comment by whit537** [comment]
Maybe?

@clone1018 Are you willing/able to own this ticket? We have a hard deadline here so we have to do *something*.

**Comment by clone1018** [comment]
I'm researching switching over my full time job's mailing system, we're using Mandrill at the moment. I can share my experience with the service we pick but my python is rusty as heck :)

**Comment by whit537** [comment]
@clone1018 M'kay. :-)

@rohitpaulk Do you have bandwidth to move us to SES before April 27?

**Comment by rohitpaulk** [comment]
Yes, I think so. Assigning this to myself

**Comment by whit537** [comment]
Awesome. :-)

!m @rohitpaulk

**Comment by yepher** [comment]
Just incase it can help here is the Mandrill migration to SparkPost [guide](https://www.sparkpost.com/mandrill-migration-guide) and this [Slack channel](http://slack.sparkpost.com) is available if you need help with the migration.

They are giving 100,000 free emails/month.

**Comment by whit537** [comment]
> They are giving 100,000 free emails/month.

@yepher "They" or "we"? ;-)

![screen shot 2016-03-02 at 3 28 58 pm](https://cloud.githubusercontent.com/assets/134455/13474343/977ec738-e08b-11e5-99a9-13f34ae326b5.png)

![screen shot 2016-03-02 at 3 29 25 pm](https://cloud.githubusercontent.com/assets/134455/13474342/977ddd1e-e08b-11e5-959b-54e50ed86417.png)

**Comment by yepher** [comment]
Yes, sorry, did not mean to seem I am not affiliated. You are right we.

**Comment by whit537** [comment]
> Hello vendors@gratipay.com,
>
This is a reminder that starting April 27, all Mandrill users will be required to have a paid monthly MailChimp account and verify ownership of all sending domains. **You can merge your MailChimp and Mandrill accounts starting today.**
>
If you missed the original announcement, [this blog post](http://blog.mandrill.com/important-changes-to-mandrill.html) explains in detail what is changing and why. If you plan to continue using Mandrill, please keep reading for instructions on how to merge your Mandrill and MailChimp accounts.
>
**What you need to do**
>
- If you haven't already done so, you'll need to add SPF and DKIM records and verify ownership of your sending domains. Here's [an article explaining how to do that](https://mandrill.zendesk.com/hc/en-us/articles/205582387-How-to-Set-up-Sending-Domains). Starting April 27, Mandrill will not send any email from unverified domains or domains without valid SPF and DKIM records, including public domains like gmail.com and yahoo.com.
>
- **If you already have a paid monthly MailChimp account**, please follow [these instructions](https://mandrill.zendesk.com/hc/en-us/articles/217600288) to merge your accounts by April 27.
>
- **If you have a free or Pay As You Go MailChimp account**, you can follow [these instructions](https://mandrill.zendesk.com/hc/en-us/articles/217600298) to merge your accounts by April 27.
>
- **If you do not have a MailChimp account**, you can follow [these instructions](https://mandrill.zendesk.com/hc/en-us/articles/218068427) to create a MailChimp account and merge it with your Mandrill account by April 27.
>
- **If you're using Mandrill as a Heroku add-on**, you'll need to create a new Mandrill account through MailChimp. The Mandrill add-on for Heroku will be removed from all Heroku apps on April 27. Find more information in our [knowledge base](https://mandrill.zendesk.com/hc/en-us/articles/218068477).
>
- If you choose not to continue using Mandrill, [here's how to export your data and close your account](https://mandrill.zendesk.com/hc/en-us/articles/217600338).
>
Any Mandrill accounts that have not been merged with a MailChimp account or closed by April 27 will be disabled.
>
If you have technical questions or need help merging your accounts, please log in to your Mandrill account and click Support in your dashboard. We’ve made support available to all users on free and paid accounts.
>
Thanks for using Mandrill, and we hope to welcome you to MailChimp soon.

https://gratipay.freshdesk.com/helpdesk/tickets/4354

**Comment by rohitpaulk** [comment]
Blocked on https://github.com/gratipay/inside.gratipay.com/issues/564

**Comment by rohitpaulk** [comment]
Blocked on https://github.com/gratipay/inside.gratipay.com/issues/585

**Comment by rohitpaulk** [comment]
I'm ticking 'Yes' on `I have a process to handle bounces and complaints`. I'll take this up, and make sure it gets done manually for now. As our volume increases, we can look into automating this

**Comment by whit537** [comment]
@rohitpaulk Sounds like https://github.com/gratipay/gratipay.com/issues/2993.

**Comment by whit537** [comment]
Once we're confident that mail is working through SES (gratipay/gratipay.com#3995) we should remove Mandrill from our DNS (SPF and DKIM).

**Comment by whit537** [comment]
Okay! Removed from SPF and DKIM removed.

**Comment by whit537** [comment]
!m @rohitpaulk
